test/e2e/wallet-helpers/index.d.ts

```diff
@@ -0,0 +1,6 @@
+import JSONRPCClient from '../../../src/lib/json-rpc-client'
```

Comment:
chore (non-blocking): can this be moved to src/index.d.ts as this now holds true for both the tests and the frontend build so would be more correct there